### DISCUSSION TITLE
Missing Marketing, Blocks and Hugo templates in Pro version google drive link

### DISCUSSION THREAD

**Discussion by rahuljaisinghani** [themesberg/flowbite#180] [General]
**Describe the bug**
I have purchased the Developer edition of Flowbite. Pro Version promised following 6 types of components
<img width="1161" alt="Screenshot 2022-06-24 at 12 45 12 PM" src="https://user-images.githubusercontent.com/25593096/175483262-089c63a6-5064-455c-a9b4-57151aea67c5.png">

Drive link which was shared contains only 2 categories. Image attached
<img width="569" alt="Screenshot 2022-06-24 at 12 46 34 PM" src="https://user-images.githubusercontent.com/25593096/175483472-311c3d93-88f3-4357-a534-903f4a803f91.png">

Please check this on an urgent basis and share the remaining modules
If this is misunderstanding from my side, please guide me to the path inside the drive document

**Comment by zoltanszogyenyi** [comment]
Hello @rahuljaisinghani,

The marketing UI code is currently available via the [Flowbite Blocks](https://flowbite.com/blocks/) system and the marketing UI pages (in the HUGO format) will arrive next week.
